appgate-server-example/config/appgate_config.yaml

```diff
@@ -3,7 +3,7 @@
 # This can be used by the Cosmos client SDK, event subscriptions, etc...
 query_node_rpc_url: tcp://poktrolld:26657
 # Pocket node URL that exposes the Cosmos gRPC service, dedicated to querying purposes.
-query_node_grpc_url: tcp://poktrolld:26658
+query_node_grpc_url: tcp://poktrolld:9090
```

Comment:
Why are we updateing `grpc` for not `normal rpc`

Reply:
@Olshansk default rpc port is `26657` so no need to change it. Default grpc port is `9090`.